### ISSUE TITLE
TS2693: 'Modal' only refers to a type, but is being used as a value here.

### ISSUE THREAD

**Issue by arty-name** [woofers/react-dialog-polyfill#20]
With the release 1.1.0 I was happy to see types declarations in the release notes. I upgraded and dropped my fake type declaration, only to see this error message from Typescript 4.6.2

 ```ts
import { Modal } from 'react-dialog-polyfill';
...
   return (
     <Modal open>....
      ^^^^^
      TS2693: 'Modal' only refers to a type, but is being used as a value here.
```

This kind of makes sense, since the export is like this:
```ts
export type Modal = React.FC<DialogProps>

**Comment by woofers** [comment]
Thanks for pointing this out @arty-name.  This is my mistake, I always forget about the `declare` syntax when typing some of my JS packages for TypeScript and was focused on fixing #8.

I have now released `v1.1.4` which should include proper support for TypeScript now.  
You can also `import type { HTMLDialog } from 'react-dialog-polyfill'` if you need to use the dialog type for anything as it looks like TypeScript does not include a full DOM type, as its not widely supported.

Let me know if this solves the issue and I can close this out.

**Comment by arty-name** [comment]
Thank you for addressing this so quickly, @woofers! I confirm that I don’t have that issue in the latest release anymore, feel free to close that ticket.
